src/ActionBuilder/ProductType/ChangeOrderOfAttributes.php

```diff
@@ -42,21 +42,33 @@ public function createUpdateActions(
         array $oldData,
         $sourceObject
     ): array {
-        $actions = [];
+        if (!class_exists(ProductTypeChangeAttributeOrderByNameAction::class, true)) {
```

Comment:
That shouldn't be necessary. I've updated the minimum commercetools sdk to the versio, which has this class.

Reply:
Hmm ok. It seems that my changes in composer json are reverted ... ok. But does `ProductTypeChangeAttributeOrderByNameAction::class` don't throw an exception if the class not exists? I think this must be a string?

`if (!class_exists('Commercetools\Core\Request\ProductTypes\Command\ProductTypeChangeAttributeOrderByNameAction, true))`

Reply:
::class resolves to a string. I didn't add this logic I just moved it to the top of the function, it was already in master